lib/Model.js

```diff
@@ -539,6 +539,11 @@ Model.update = function(NewModel, key, update, options, next) {
     options.updateTimestamps = true;
   }
 
+  // default updateExpires to true
```

Comment:
This should be `false`, correct? In this comment? Although it's just a comment, probably a good idea to ensure it is correct.